lib/matplotlib/rcsetup.py

```diff
@@ -501,6 +501,8 @@ def __call__(self, s):
 
 # a map from key -> value, converter
 defaultParams = {
+    'style': ['', validate_stringlist],
```

Comment:
On the top on being able to create stylesheets by "cascading" them (it's a very small implementation of cascading...), you could have a style in your matplotlibrc so that you don't have to call the `style.use` function every time you create a new script.